modules/data.remote/inst/s2ndvi.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Module to calculate NDVI from sentinel 2 data, based on Satellitetools by Olli Nevalainen.
+This will be used to create automated workflows for the remote data module.
+
+Warning: Point coordinates as input has currently not been implemented.
+
+Author: Ayush Prasad
+"""
+
+from satellitetools import gee
```

Comment:
I'm not familiar with python, but just to confirm, does this also work without having satellitetools as a library?

Reply:
satellitetools is just the name of the folder which contains the original script `gee.py`, this script is required.

Reply:
Yup that much I followed, I wasn't just aware if one can simply import a function from a folder